cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreManager.java

```diff
@@ -333,7 +333,7 @@ private Datastore createDatastore(String dbName, KeyProvider provider) throws Da
             // if it does not exist
 
             //Pass database directory, database name, and SQLCipher key provider
-            DatastoreImpl ds = new DatastoreImpl(dbDirectory, dbName, provider);
+            QueryableDatastore ds = new QueryableDatastore(dbDirectory, dbName, provider);
```

Comment:
I'm also troubled by this:

```LargeAttachmentDatastore (being the last set of function we added), but they wouldn't know they'd just have the new LargeAttachmentDatastore methods appear```

Because the name of each class because an increasing blatant lie (it's actually a Large Attachment _and_ Queryable Datastore)